README.md

```diff
@@ -167,10 +166,15 @@ A recent update  reduces the voltage supply from 3.3V to 2.7V  during sleep, the
 
```

Comment:
@jeffmer How can we measure the light sleep current of V3?

Reply:
To measure it, I took out the battery and connected some thin wires to the battery connectors on the watch and then used an external lipo battery in series with a multimeter to measure the current.

Reply:
Is the mAmp value appearing in the "battery" app reliable? My V3 displays something between 20 and 45 mA btw

Reply:
I don’t think it is very accurate but it does increase if you turn up brightness or CPU frequency so it’s a useful indicator.

Reply:
Indeed. Btw my watch battery died during the night. It was at 4.13V at 7pm and dead when I checked at 2pm today.

ps : sorry if I added the commits from the other branch to this branch, I'm not always super comfortable with git and github so yeah